### PR TITLE
Add jest

### PR DESCRIPTION
**Pull request by fabianchua6**
Installing jest and also configuring it to work with typescript

### CODE REVIEW

**Review by glencbz — CHANGES_REQUESTED** [review]
There doesn't seem to be any jest here

**Comment by stonefruit** [comment]
hmm. is this just adding jest? or also the test for the FIN ?

**Comment by fabianchua6** [comment]
Yes it's only for adding jest, without the test for the fin. Should I include it in too?

File: backend/src/util/VerifyFIN.test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
```
Comment (glencbz) [inline backend/src/util/VerifyFIN.test.ts:1]: Why `eslint-disable`?

File: backend/src/util/VerifyFIN.test.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import validateFIN from "./VerifyFIN"
+
+test('Valid FIN', () => {
+    expect(validateFIN("F1234567N")).toBeTruthy
+})
+
+test('Lowercase first alphabet', () => {
+    expect(validateFIN("f3725759K")).toBeTruthy
```
Comment (glencbz) [inline backend/src/util/VerifyFIN.test.ts:9]: The assertions need to be called, e.g. `toBeTruthy()`.

File: backend/src/util/VerifyFIN.test.ts (hunk 1 above)
Comment (fabianchua6) [reply backend/src/util/VerifyFIN.test.ts:1]: Thought it was a workaround for using tests files but I managed to find another method which adds jest env to eslint, makes much more sense I believe. Thanks for pointing this out!

File: backend/src/util/VerifyFIN.test.ts (hunk 2 above)
Comment (fabianchua6) [reply backend/src/util/VerifyFIN.test.ts:9]: Got it!

**Review by glencbz — APPROVED** [review]
Awesome stuff. Remember to resolve the merge conflicts.

**Comment by fabianchua6** [comment]
Finally... that felt good

**Review by glencbz — CHANGES_REQUESTED** [review]
Did you delete frontend/package-lock.json?

**Comment by glencbz** [comment]
There were merge conflicts, so I fixed this out-of-band. Good work @fabianchua6!

**Pull request closed without merging**